bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/i18n/deconz.properties

```diff
@@ -0,0 +1,189 @@
+thing-type.config.deconz.bridge.apikey.description = If no API Key is provided, a new one will be requested. You need to authorize the access on the deCONZ web interface.
+thing-type.config.deconz.bridge.host.label = Host Address
+thing-type.config.deconz.bridge.host.description = IP address or host name of deCONZ interface.
+thing-type.config.deconz.bridge.httpPort.label = HTTP Port
+thing-type.config.deconz.bridge.httpPort.description = Port of the deCONZ HTTP interface.
+thing-type.config.deconz.bridge.port.label = Websocket Port
+thing-type.config.deconz.bridge.port.description = Port of the deCONZ Websocket.
+thing-type.config.deconz.bridge.timeout.label = Timeout
+thing-type.config.deconz.bridge.timeout.description = Timeout for asynchronous HTTP requests (in milliseconds).
+thing-type.config.deconz.colorlight.colormode.label = Color Mode
+thing-type.config.deconz.colorlight.colormode.description = Override the default color mode (auto-detect)
+thing-type.config.deconz.colorlight.colormode.option.hs = HSB
+thing-type.config.deconz.colorlight.colormode.option.xy = XY
+thing-type.config.deconz.colorlight.id.label = Device ID
+thing-type.config.deconz.colorlight.id.description = The deCONZ bridge assigns an integer number ID to each device.
+thing-type.config.deconz.colorlight.transitiontime.label = Transition Time
+thing-type.config.deconz.colorlight.transitiontime.description = Time to move between two states. If empty, the default of the device is used. Resolution is 1/10 second.
+thing-type.config.deconz.light.id.label = Device ID
+thing-type.config.deconz.light.id.description = The deCONZ bridge assigns an integer number ID to each device.
+thing-type.config.deconz.light.transitiontime.label = Transition Time
+thing-type.config.deconz.light.transitiontime.description = Time to move between two states. If empty, the default of the device is used. Resolution is 1/10 second.
+thing-type.config.deconz.lightgroup.colormode.label = Color Mode
+thing-type.config.deconz.lightgroup.colormode.description = Override the default color mode (auto-detect)
+thing-type.config.deconz.lightgroup.colormode.option.hs = HSB
+thing-type.config.deconz.lightgroup.colormode.option.xy = XY
+thing-type.config.deconz.lightgroup.id.label = Device ID
+thing-type.config.deconz.lightgroup.id.description = The deCONZ bridge assigns an integer number ID to each group.
+thing-type.config.deconz.sensor.id.label = Device ID
+thing-type.config.deconz.sensor.id.description = The deCONZ bridge assigns an integer number ID to each device.
+thing-type.config.deconz.sensor.lastSeenPolling.label = LastSeen Poll Interval
+thing-type.config.deconz.sensor.lastSeenPolling.description = Interval to poll the deCONZ Gateway for this sensor's "last_seen" channel. Polling is disabled when set to 0 (default: 1440, once per day).
+
+# channel types
+
+channel-type.deconz.alarm.label = Alarm
+channel-type.deconz.alarm.description = Alarm was triggered.
+channel-type.deconz.alert.label = Alert
```

Comment:
@wborn Your i18n-tool does not yet consider command options. The channel-type definition for the `alert` looks like this:

https://github.com/openhab/openhab-addons/blob/062f45440663b431d6213d2854c17027787c9269/bundles/org.openhab.binding.deconz/src/main/resources/OH-INF/thing/light-thing-types.xml#L185-L196